ecdsa/Cargo.toml

```diff
@@ -15,17 +15,17 @@ edition = "2021"
 rust-version = "1.56"
 
 [dependencies]
-elliptic-curve = { version = "0.11.6", default-features = false, features = ["sec1"] }
-signature = { version = ">= 1.3.1, <1.5", default-features = false, features = ["rand-preview"] }
+elliptic-curve = { git = "https://github.com/RustCrypto/traits", version = "0.12.0-pre", default-features = false, features = ["sec1"] }
+signature = { version = "1.5", default-features = false, features = ["rand-preview"] }
 
 # optional dependencies
 der = { version = "0.5", optional = true }
-rfc6979 = { version = "0.1", optional = true, path = "../rfc6979" }
+rfc6979 = { git = "https://github.com/khonsulabs/signatures", branch = "elliptic-curve-digest", optional = true }
```

Comment:
```suggestion
rfc6979 = { version = "0.1", optional = true, path = "../rfc6979" }
```
This needs to be done when this is ready to be merged.

Reply:
Not sure why you had to change this at all?

Reply:
Same reason as above, so I can depend on it without `[patch]`.

Reply:
But it's a crate in the same repository referred to by `path` already?

Reply:
Anyway I can't merge this until you change it back...

Reply:
Yes, but when you depend on `ecdsa` as a git dependency it will still pull `rfc6979` in from crates.io. I think?

Reply:
And this is where `[patch.crates-io]` is helpful.

Can you change it back and then figure out what to do downstream?

Reply:
I can do that, no problem.

Reply:
Done.